prov/verbs/src/verbs_mr.c

```diff
@@ -377,6 +409,13 @@ static int vrb_mr_regattr(struct fid *fid, const struct fi_mr_attr *attr,
 			   domain->util_domain.info_domain_caps, attr,
 			   &cur_abi_attr);
 
+	if (flags & FI_MR_DMABUF) {
+		return vrb_reg_dmabuf(domain, attr->dmabuf->fd,
+				      attr->dmabuf->offset, attr->dmabuf->len,
+				      vrb_mr_ofi2ibv_access(attr->access, domain),
+				      mr);
```

Comment:
This bypasses the reg cache.

Reply:
That was intentional because I don't know how we deal with a dma-buf in the cache or if we need to.

Reply:
Good for now. In the long run, we may still want the registration be cached, probably by using a separate mr cache that use <fd, offset> instead of addr as the key.

Reply:
The problem is that I don't know how you verify that the entry is still valid or eject it from the cache.